src/components/MoneyReportHeader.tsx

```diff
@@ -62,6 +64,14 @@ function MoneyReportHeader({session, policy, chatReport, nextStep, report: money
           isPolicyAdmin && (isApproved || isManager)
         : isPolicyAdmin || (ReportUtils.isMoneyRequestReport(moneyRequestReport) && isManager);
     const isDraft = ReportUtils.isDraftExpenseReport(moneyRequestReport);
+    const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
+
+    const cancelPayment = useCallback(() => {
+        // @ts-expect-error TODO: Remove this once IOU (https://github.com/Expensify/App/issues/24926) is migrated to TypeScript.
```

Comment:
I think this has happened, see https://github.com/Expensify/App/pull/34092. Is it possible to get rid of this?